usdSchemas/GafferCycles.usda

```diff
@@ -0,0 +1,130 @@
+#usda 1.0
+(
+	subLayers = [
+		@usdLux/schema.usda@,
+		@usd/schema.usda@
+	]
+)
+
+over "GLOBAL" (
+	customData = {
+		string libraryName = "GafferCycles"
+		bool skipCodeGeneration = 1
+		bool useLiteralIdentifier = 1
+	}
+)
+{
+}
+
+# Here we define a bunch of codeless auto-apply API schemas for extending the
+# standard UsdLux lights with inputs specific to Cycles. This approach is
+# modelled on the one used by UsdRiPxr to add RenderMan-specific inputs, and
+# we believe is the one Pixar intends everyone to use.
+
+class "GafferCyclesLightAPI" (
+	customData = {
+		token[] apiSchemaAutoApplyTo = [ "DistantLight", "DiskLight", "DomeLight", "RectLight", "SphereLight" ]
+		string apiSchemaType = "singleApply"
+		string className = "GafferCyclesLightAPI"
+	}
+	inherits = </APISchemaBase>
+)
+{
+
+	string inputs:cycles:lightgroup = "" (
+		displayGroup = "Basic"
+		displayName = "Light Group (Cycles)"
+	)
+
+	bool inputs:cycles:use_mis = true (
+		displayGroup = "Refine"
+		displayName = "Multiple Importance (Cycles)"
```

Comment:
This and several other labels are too wide for the NodeEditor in Gaffer. Could we shorten them to fit while keeping them intelligible?

Reply:
Done, maybe a little vague but it isn't too much different to Arnold now.